packages/store/reducers.js

```diff
@@ -49,4 +49,6 @@ export default combineReducers({
   closeAccount: closeAccountReducer,
   productFeatures: productFeaturesReducer,
   drafts: draftsReducer,
+  generalSettings: GeneralSettingsReducer,
```

Comment:
therefore there would likely also benefit from being lowercase too
```suggestion
  generalSettings: generalSettingsReducer,
```